### ISSUE TITLE
[BUG] Unable to like a post

### ISSUE THREAD

**Issue by fitittome** [vitorpamplona/amethyst#1225]
Tapping the heart icon to like a post displays a list of emojees, none selectable.

**To Reproduce**
1. Tap heart icon.

**Expected behavior**
I'd expect the heart icon to illuminate.

![IMG_20241216_004516](https://github.com/user-attachments/assets/0fd245d3-c002-48b7-a7b5-0138505914f2)

**Device (please complete the following information):**
 - Pixel 8
 - GrapheneOS (Android 15)
 - App Version FDroid 0.91.0

**Bounty (in Bitcoin sats) offered for a solution**
None

**Comment by davotoula** [comment]
Can you post a screen recording please?

**Comment by fitittome** [comment]
> Can you post a screen recording please?

Sure.  Here I'm tapping on the heart.

https://github.com/user-attachments/assets/a5e9964b-a8dd-4d41-80ff-86e6256b36c6

**Comment by davotoula** [comment]
It looks like your 👀 reaction registers?

Is it a heart you are looking for?

**Comment by fitittome** [comment]
> It looks like your 👀 reaction registers?
> 
> Is it a heart you are looking for?

Yes!  I was expecting a tap on the heart would illuminate it.  A press to pop-up the emojis.  But both give the emoji pop-up.

**Comment by davotoula** [comment]
Nostr supports multiple reactions. You can click multiple icons and they will all be registered. You can also deselect by clicking it again.

If you want a heart you can add it by clicking the emoji+ icon, search for a heart icon via your keyboard, hit add and hit save.

You can even add custom emoji packs to your nostr profile via:

https://emojito.meme/

**Comment by davotoula** [comment]
Any updates?

**Comment by vitorpamplona** [comment]
When the popup shows up on Amethyst, you can click on the "+" face and add a heart to your list of reactions. You can also remove all the others and if it is only one reaction there, the popup won't show and the click will simply light up the icon. 

Hearts tend to be quite meaningless on Nostr. We encourage people to use other emojis to more precisely demonstrate their emotions towards a post :)

**Comment by fitittome** [comment]
Yes, I get that.  I use Akkoma on the Fedi and expressive emoji reactions work in a similar way.  But the heart isn't meaningless.  It serves a similar  purpose on both platforms. It's a little sign of thanks, a confidence boost, a mark of acknowledgment, even though it serves no algorithmic purpose.  But it's OK to differ on this, I'm clearly biased by the apps I use elsewhere.

**Comment by davotoula** [comment]
But you are aware you can add a hear easily right?

![Screenshot_20250102-155218.png](https://github.com/user-attachments/assets/714b6a18-6586-4bf6-a6ca-008796d34c34)

**Comment by fitittome** [comment]
Yes, you explained the process to add it to the list of default emojis earlier in the thread....  and Victor has explained how to remove all the other default emojis so that the popup doesn't show at all and therefore becomes the default when the heart icon is tapped.

**Comment by davotoula** [comment]
As designed
